Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.cpp

```diff
@@ -1352,8 +1363,12 @@ void AXIsolatedTree::processQueuedNodeUpdates()
 #if ENABLE(AX_THREAD_TEXT_APIS)
 AXTextMarker AXIsolatedTree::firstMarker()
 {
-    RefPtr root = rootNode();
-    return root ? AXTextMarker { root->treeID(), root->objectID(), 0 } : AXTextMarker();
+    ASSERT(!isMainThread());
+    // The first marker should be constructed from the root WebArea, not the true root of the tree
+    // which is the ScrollView, so that when we convert the marker to a CharacterPosition, there
+    // is an associated node. Otherwise, the CharacterPosition will be null.
+    RefPtr rootWebArea = webArea();
```

Comment:
Thought that the method is called rootWebArea.

Reply:
Updated, thanks for catching this!